node/src/Channel.ts

```diff
@@ -275,6 +288,80 @@ export class Channel extends EnhancedEventEmitter
 		});
 	}
 
+	async requestBinary(
+		method: Method,
+		bodyType?: RequestBody,
+		bodyOffset?: number,
+		handlerId?: string): Promise<Response>
+	{
+		this.#nextId < 4294967295 ? ++this.#nextId : (this.#nextId = 1);
+
+		const id = this.#nextId;
+
+		logger.error('request() [method:%s, id:%s]', id);
+
+		if (this.#closed)
```

Comment:
Why are we checking closed state that late in this method? It can be done at the top. Did we do it before?

Reply:
I asked that myself too. Changing in both `request` and `requestBinary`.